app/code/Magento/WishlistGraphQl/etc/schema.graphqls

```diff
@@ -5,6 +5,10 @@ type Query {
     wishlist: WishlistOutput @resolver(class: "\\Magento\\WishlistGraphQl\\Model\\Resolver\\WishlistResolver") @doc(description: "The wishlist query returns the contents of a customer's wish list") @cache(cacheable: false)
 }
 
+type Mutation {
+    removeProductsFromWishlist(input: [Int]!): WishlistOutput @resolver(class: "\\Magento\\WishlistGraphQl\\Model\\Resolver\\WishlistRemoveItem") @doc(description: "Delete a list of wishlist items")
```

Comment:
As per design document, it should be: https://github.com/magento/architecture/pull/326
```suggestion
    removeProductsFromWishlist(input: RemoveProductsFromWishlistInput): RemoveProductsFromWishlistOutput @resolver(class: "\\Magento\\WishlistGraphQl\\Model\\Resolver\\WishlistRemoveItem") @doc(description: "Delete a list of wishlist items")
```